frontend/src/components/RecipeCard/RecipeCard.tsx

```diff
@@ -14,9 +19,33 @@ export default function RecipeCard(props: Readonly<RecipeCardProps>) {
     }
 
     return (
-        <button className="recipe-card" onClick={() => (goToRecipeDetailsPage(props.recipe.id))}>
-            <h3>{props.recipe.name}</h3>
-            <p>{props.recipe.description}</p>
-        </button>
-    )
+        <Card className="recipe-card">
+            <CardActionArea>
+                <CardMedia
+                    component="img"
+                    height="140"
+                    image= {props.recipe.imageUrl}
+                    alt= "https://res.cloudinary.com/dergdq6qs/image/upload/v1712181382/o5gm3luzvcnv0ifsznnw.jpg"
```

Comment:
what is this alt text? you could say as alt text the recipe name for example, but not linking an image

Reply:
I want to make an alternative image. I didn't know for now how to do it, so I kept the Link of the alternative image in the alt, not to lose it :)

Reply:
if you want an alternative image you can use the ternary operator: `props.recipe.imageUrl ? your-card-media-with-image : your-alternative-card-media-with-alternative-image`

This is not a good practice and if you want to save it for later, you should have a document where you can save things like this

Reply:
the alt-attribute does not replace your image. its an alternative if the image can not be loaded for some reason. than there will be a text